### PR TITLE
Fix snapshot publish own module

### PR DESCRIPTION
**Pull request by ThisaruGuruge**
## Purpose
$subject

Fixes #<Issue Number>

## Check List 
- [ ] Read the [Contributing Guide](https://github.com/ballerina-platform/ballerina-lang/blob/master/CONTRIBUTING.md)
- [ ] Updated Change Log
- [ ] Checked Tooling Support (#<Issue Number>)
- [ ] Added necessary tests
  - [ ] Unit Tests
  - [ ] Spec Conformance Tests
  - [ ] Integration Tests
  - [ ] Ballerina By Example Tests
- [ ] Increased Test Coverage   
- [ ] Added necessary documentation  
  - [ ] API documentation 
  - [ ] Module documentation in Module.md files
  - [ ] Ballerina By Examples

### CODE REVIEW

File: stdlib/release/src/commons/functions.bal [hunk 1]
```diff
@@ -91,14 +95,15 @@ public function logAndPanicError(string message, error e) {
...
-        log:printInfo(moduleString);
+    foreach Module module in modules {
+        printModule(module);
     }
 }
 
+public function printModule (Module module) {
+    log:printInfo(module.name + " " + module.'version);
+}
+
```
Comment (niveathika) [inline stdlib/release/src/commons/functions.bal:106]: Any reason to have a separate function? It's just a log isnt it?

**Comment by niveathika** [comment]
@ThisaruGuruge Could we also check if the dependent module are triggered correctly, Noticed only one of the dependants is triggered correctly

File: stdlib/release/src/commons/functions.bal (hunk 1 above)
Comment (ThisaruGuruge) [reply stdlib/release/src/commons/functions.bal:106]: Just for the convenience. To use in any place with the module name and the version.

**Comment by ThisaruGuruge** [comment]
> @ThisaruGuruge Could we also check if the dependent module are triggered correctly, Noticed only one of the dependants is triggered correctly

Yes, there was an issue. It is also fixed with this.

File: stdlib/release/src/commons/functions.bal (hunk 1 above)
Comment (niveathika) [reply stdlib/release/src/commons/functions.bal:106]: Ack

**Review by niveathika — APPROVED** [review]

**Pull request merged**